.packit.yaml

```diff
@@ -13,198 +13,142 @@ srpm_build_deps:
-          - hatch build -t sdist
-          - bash -c "ls dist/tmt-*.tar.gz"
-        get-current-version:
-          # Use `dev888` instead of the last version digit to make
-          # builds older than the final release and newer than
-          # copr builds from main. Remove the remaining suffix
-          # generated by `hatch version` as well so that build
-          # with the latest timestamp always wins.
-          - bash -c "hatch version | sed -E 's/\\.[0-9]+\\.dev.*/.dev888/'"
-
-  # Test pull requests (core)
-  - job: tests
-    identifier: core
+      <<: *base-actions
+      get-current-version:
+        # Use `dev888` instead of the last version digit to make
+        # builds older than the final release and newer than
+        # copr builds from main. Remove the remaining suffix
+        # generated by `hatch version` as well so that build
+        # with the latest timestamp always wins.
+        - bash -c "hatch version | sed -E 's/\\.[0-9]+\\.dev.*/.dev888/'"
+  - &tests
+    job: tests
     trigger: pull_request
-    targets:
+    targets: &latest-targets
       - fedora-latest-stable
       - epel-9
+    identifier: core
     tf_extra_params:
-        test:
-            tmt:
-                name: /plans/features/(core|basic)
-
-  # Test pull requests (full)
-  # Do not run extended unit tests, that plan gets its own job because
-  # of podman vs systemd-resolved flakiness.
-  - job: tests
+      test:
+        tmt:
+          name: /plans/features/(core|basic)
+  - # Test pull requests (full)
+    # Do not run extended unit tests, that plan gets its own job because
+    # of podman vs systemd-resolved flakiness.
+    <<: *tests
+    targets: *all-targets
+    require: &require-full-tests
+      label:
+        present:
+          - full test
+        absent:
+          - discuss
     identifier: full
-    trigger: pull_request
-    targets:
-      - fedora-all
-      - epel-9
-    require:
-        label:
-          present:
-            - full test
-          absent:
-            - discuss
     tf_extra_params:
-        test:
-            tmt:
-                name: '^(?!/plans/features/extended-unit-tests).*$'
-
-  - job: tests
+      test:
+        tmt:
+          name: '^(?!/plans/features/extended-unit-tests).*$'
+  - <<: *tests
+    targets: [ fedora-latest-stable ]
+    require: *require-full-tests
     identifier: extended-unit-tests
-    trigger: pull_request
-    targets:
-      - fedora-latest-stable
-    require:
-        label:
-          present:
-            - full test
-          absent:
-            - discuss
     tf_extra_params:
-        test:
-            tmt:
-                name: '/plans/features/extended-unit-tests$'
-
-  # Test pull requests (provision)
-  - job: tests
-    identifier: provision
-    trigger: pull_request
+      test:
+        tmt:
+          name: '/plans/features/extended-unit-tests$'
+  - # Test pull requests (provision)
+    <<: *tests
     use_internal_tf: True
```

Comment:
I guess we should apply the `BusinessUnit: tmt` part for this internal job as well? Seems we forgot it here.

Reply:
Ok, added in d710a8ec.

Reply:
Thanks for that :)